net/src/ipc_net_worker.rs

```diff
@@ -327,6 +322,7 @@ impl IpcNetWorker {
             }"#;
 }
 
+/*
```

Comment:
why are we doing this?

if the tests aren't relevant at all then let's delete them

if they are relevant but need fixing, use the "broken-tests" compiler attribute to elide them

Reply:
went ahead and deleted, it'll be easier to rewrite from scratch when I take that pass.